### DISCUSSION TITLE
Argocd syncs for a value that does not exist?

### DISCUSSION THREAD

**Discussion by sjoh0704** [argoproj/argo-cd#11383] [Q&A]
Hi,
I wonder if argocd syncs for a value that does not exist.

For example, 
I deployed prometheus deployment without annotations.test=test through argocd.
And then using kubectl cli, I attached annotations.test=test to prometheus deployment.
At this time, out of sync did not occur in the argocd dashboard.

Is this right? I'm using argocd v2.2.5.
I think argocd doesn't sync values that don't exist in repository, so I wonder if this is right.

I'd appreciate it if you let me know.

**Comment by kostis-codefresh** [comment]
What happens if you press the "refresh/hard refresh" button in the Argo CD UI for the prometheus application?
